infra/terraform/modules/hash_application/type_fetcher.tf

```diff
@@ -0,0 +1,51 @@
+locals {
+  type_fetcher_service_name = "typefetcher"
+  type_fetcher_prefix       = "${var.prefix}-${local.type_fetcher_service_name}"
+  type_fetcher_param_prefix = "${local.param_prefix}/${local.type_fetcher_service_name}"
+}
+
+resource "aws_ssm_parameter" "type_fetcher_env_vars" {
+  # Only put secrets into SSM
+  for_each = { for env_var in var.type_fetcher_env_vars : env_var.name => env_var if env_var.secret }
+
+  name = "${local.type_fetcher_param_prefix}/${each.value.name}"
+  # Still supports non-secret values
+  type      = each.value.secret ? "SecureString" : "String"
+  value     = each.value.secret ? sensitive(each.value.value) : each.value.value
+  overwrite = true
+  tags      = {}
+}
+
+locals {
+  type_fetcher_service_container_def = {
+    name        = "${local.type_fetcher_prefix}container"
+    image       = "${var.type_fetcher_image.url}:latest"
+    cpu         = 0 # let ECS divvy up the available CPU
+    mountPoints = []
+    volumesFrom = []
+    command     = ["type-fetcher"]
+    healthCheck = {
+      command  = ["CMD", "/hash-graph", "type-fetcher", "--healthcheck"]
+      retries  = 5
+      interval = 20
+      timeout  = 5
+
+    }
+    logConfiguration = {
+      logDriver = "awslogs"
+      options = {
+        "awslogs-create-group"  = "true"
+        "awslogs-group"         = local.log_group_name
+        "awslogs-stream-prefix" = local.type_fetcher_service_name
+        "awslogs-region"        = var.region
```

Comment:
Logs haven't been published in prod yet, not sure what's happening. We may want to keep an eye on that. Perhaps it's because of the new `scratch` images? Seems like the logs are just not submitted to aws

Reply:
This could be true. Is it possible for `aws` to mount a volume and `tail -f` that? Otherwise, we may want to register a logger via `tracing`, which can be used from `aws`. For a temporary solution we could use `alpine` again.

Reply:
Yeah i think we can using Amazon EFS volumes, but we already use the Cloudwatch log collector which relies on stdout. We should look into this when we get a chance